v2/periphery/LockupLinearBatchStreamCreator.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.13;
+
+import { IAllowanceTransfer } from "@permit2/interfaces/IAllowanceTransfer.sol";
+import { IPRBProxy } from "@prb/proxy/interfaces/IPRBProxy.sol";
+import { IPRBProxyRegistry } from "@prb/proxy/interfaces/IPRBProxyRegistry.sol";
+import { ISablierV2LockupLinear } from "@sablier/v2-core/interfaces/ISablierV2LockupLinear.sol";
+import { ud60x18 } from "@sablier/v2-core/types/Math.sol";
+import { IERC20 } from "@sablier/v2-core/types/Tokens.sol";
+import { ISablierV2ProxyTarget } from "@sablier/v2-periphery/interfaces/ISablierV2ProxyTarget.sol";
+import { Batch, Broker, LockupLinear, Permit2Params } from "@sablier/v2-periphery/types/DataTypes.sol";
```

Comment:
Why would we add one more line of code?

Reply:
Because this is an example and clarity is more important than marginal numbers of lines of code (it's really just 1 line).

The `LockupLinear` is defined in `@sablier/v2-core` - that's where we should present it in a public example.

Reply:
Resolved in the last commit